tools/ci_build/github/azure-pipelines/centos6-gpu-ci-pipeline.yml

```diff
@@ -0,0 +1,23 @@
+jobs:
+- job: linux_centos_ci
+  timeoutInMinutes:  60
+  pool:
+    vmImage: 'ubuntu-latest'
+  strategy:
+    maxParallel: 2
+    matrix: 
+      Debug:
+        BuildType: Debug
+      Release:
+        BuildType: Release
+  steps:
+    - task: CmdLine@2
+      inputs:
+        script: |
+          docker build -t onnxruntime-centos6gpu --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=3.6 -f Dockerfile.centos6_gpu .
+        workingDirectory: $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker
+    - task: CmdLine@2
+      inputs:
+        script: |
+          docker run --rm --volume $(Build.SourcesDirectory):/onnxruntime_src --volume $(Build.BinariesDirectory):/build -e NIGHTLY_BUILD onnxruntime-centos6gpu /usr/bin/python36 /onnxruntime_src/tools/ci_build/build.py --build_dir /build --config $(BuildType) --skip_submodule_sync  --parallel --build_shared_lib --use_openmp --cmake_path /usr/bin/cmake --ctest_path /usr/bin/ctest --use_automl --build_wheel
+        workingDirectory: $(Build.SourcesDirectory)
```

Comment:
why do you need to update the run_dockerbuild.sh which is not used for your case? Please refer to other linux build pipeline.

Reply:
Not sure I understand the comment.

---
In reply to: [336322149](https://github.com/microsoft/onnxruntime/pull/2165#discussion_r336322149) [](ancestors = 336322149)